### DISCUSSION TITLE
smartedit only have two label background and spleen

### DISCUSSION THREAD

**Discussion by lg2578** [Project-MONAI/MONAILabel#871] [Q&A] [answered]
I've tried task09_spleen ,and task10_colon ,and my own datasets,
the smartedit panel's label  droplist only show spleen 
then how to add more labels

**Comment by diazandr3s** [comment, marked as answer]
Hi @lg2578,

Thanks for opening this discussion and for using MONAI Label.
The reason you only see those two labels (i.e. background and spleen) is that you're using the default model - a model that we used for demonstration purposes. If you upgrade the MONAI Label version (0.4.1), the default DeepEdit model has more labels. Here is the list:

https://github.com/Project-MONAI/MONAILabel/blob/main/sample-apps/radiology/lib/configs/deepedit.py#L46

If you want to add/delete/change the labels, please follow the instruction given here: https://youtu.be/wtiEe_jiUzg?t=2840

Hope this helps

**Reply by lg2578** [reply]
thanks very much
